python/paddle/fluid/contrib/slim/tests/quantization/mkldnn_int8_quantization_strategy.py

```diff
@@ -0,0 +1,212 @@
+# see the license for the specific language governing permissions and
+# limitations under the license.
+
+import unittest
+import os
+import sys
+import argparse
+import shutil
+import logging
+import struct
+import six
+import numpy as np
+import paddle
+import paddle.fluid as fluid
+from paddle.fluid.contrib.slim.core import Compressor
+
+logging.basicConfig(format='%(asctime)s-%(levelname)s: %(message)s')
+_logger = logging.getLogger(__name__)
+_logger.setLevel(logging.INFO)
+
+
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--batch_size', type=int, default=1, help='batch size')
+    parser.add_argument(
+        '--infer_model', type=str, default='', help='model path')
+    parser.add_argument('--infer_data', type=str, default='', help='data file')
+    parser.add_argument(
+        '--int8_model_save_path',
+        type=str,
+        default='./output',
+        help='path to save int8 model with fp32 weights')
+    parser.add_argument(
+        '--warmup_batch_size',
+        type=int,
+        default=100,
+        help='batch size for quantization warmup')
+    parser.add_argument(
+        '--accuracy_diff_threshold',
+        type=float,
+        default=0.01,
+        help='Accepted accuracy drop threshold.')
+
+    test_args, args = parser.parse_known_args(namespace=unittest)
+
+    return test_args, sys.argv[:1] + args
+
+
+class TestMKLDNNPostTrainingQuantStrategy(unittest.TestCase):
+    """
+    Test API of Post Training quantization strategy for int8 with MKLDNN.
+    """
+
+    def _reader_creator(self, data_file='data.bin', cycle=False):
+        def reader():
+            with open(data_file, 'rb') as fp:
+                num = fp.read(8)
+                num = struct.unpack('q', num)[0]
+                imgs_offset = 8
+                img_ch = 3
+                img_w = 224
+                img_h = 224
+                img_pixel_size = 4
+                img_size = img_ch * img_h * img_w * img_pixel_size
+                label_size = 8
+                labels_offset = imgs_offset + num * img_size
+                step = 0
+
+                while step < num:
+                    fp.seek(imgs_offset + img_size * step)
+                    img = fp.read(img_size)
+                    img = struct.unpack_from('{}f'.format(img_ch * img_w *
+                                                          img_h), img)
+                    img = np.array(img)
+                    img.shape = (img_ch, img_w, img_h)
+                    fp.seek(labels_offset + label_size * step)
+                    label = fp.read(label_size)
+                    label = struct.unpack('q', label)[0]
+                    yield img, int(label)
+                    step += 1
+                    if cycle and step == num:
+                        step = 0
+
+        return reader
+
+    def _update_config_file(self, fp32_model_path, output_path):
+        config_path = './quantization/config_mkldnn_int8.yaml'
```

Comment:
Do you run this unit test file locally? If this file in `quantization` directory,  I don't think this UT can be run successfully because of the `config_path` value. When you raise a pull request, please be sure that its UT can be run successfully in your local machine.

Reply:
It runs ok with cmd `ctest -R test_slim_int8_*` on my local machine.